src/encord_active/app/data_quality/sub_pages/explorer.py

```diff
@@ -259,13 +253,13 @@ def fill_data_quality_window(current_df: pd.DataFrame):
     else:
         cols: List = []
         similarity_expanders = []
-        for i, (row_no, row) in enumerate(paginated_subset.iterrows()):
+        for i, (_, row) in enumerate(paginated_subset.iterrows()):
```

Comment:
Is `i` and `_` not the same thing?

Reply:
They are, addressed in our big refactor 👍